autoupdate.services/src/org/netbeans/api/autoupdate/InstallSupport.java

```diff
@@ -120,7 +120,7 @@ public Installer doValidate(Validator validator, ProgressHandle progress/*or nul
      * @see #doValidate
      */
     public Restarter doInstall(Installer installer ,ProgressHandle progress/*or null*/) throws OperationException {
-        Boolean restart = impl.doInstall (installer, progress, false);
```

Comment:
Why force install ? This will, I think, disable restart for all cases especially when some module indicates the platform needs to be restarted (`needsrestart` in autoupdate xml catalog) ?

Reply:
The change was made to make the PR functional. In the PR overview there is a note:

* module isn't installed immediately after download - workarounded as 8956f83
    * future version of the NBM shouldn't require restart

E.g. this actual change should not get merged at the end.

Reply:
Understood, but does this affect just javac, or all modules installed manually using AU ?